.travis/yapf.sh

```diff
@@ -9,17 +9,49 @@ builtin cd "$(dirname "${BASH_SOURCE:-$0}")"
 ROOT="$(git rev-parse --show-toplevel)"
 builtin cd "$ROOT"
 
-yapf \
-    --style "$ROOT/.style.yapf" \
-    --in-place --recursive --parallel \
-    --exclude 'python/ray/cloudpickle' \
-    --exclude 'python/ray/dataframe' \
-    --exclude 'python/ray/rllib' \
-    -- \
-    'test' 'python'
+YAPF_FLAGS="--style $ROOT/.style.yapf --in-place --recursive --parallel"
+
+YAPF_EXCLUDES="--exclude 'python/ray/cloudpickle' \
+--exclude 'python/ray/dataframe' \
+--exclude 'python/ray/rllib' \
+--exclude 'python/build' \
+--exclude 'python/ray/pyarrow_files' \
+--exclude 'python/ray/core/src/ray/gcs' \
+--exclude 'python/ray/common/thirdparty'"
+
+UPSTREAM_MASTER=${RAY_UPSTREAM_BRANCH:-upstream/master}
+
+# Format specified files
+format() {
+    yapf $YAPF_FLAGS -- "$@"
+}
+
+# Format files that differ from main branch
+format_changed() {
+    FILES=`git diff --name-only $UPSTREAM_MASTER -- '*.py'`
+    yapf $YAPF_FLAGS $YAPF_EXCLUDES -- $FILES
```

Comment:
This function should not exclude anything, since if you want that sort of fine-grained control over what you format, you can look at the output from `git status` manually and just pass those files to `format`. This is meant to be a quick way to format all files that differ from the main branch.

Reply:
If you want to do that, you can run with --files. Consider the common use case. Would a user be surprised if the auto-formatter formatted a file that is excluded?